docs/paper/dev/api/commands-api/arguments.mdx

```diff
@@ -0,0 +1,186 @@
+---
+slug: /dev/commands-api/arguments
```

Comment:
Not sure that I like these slugs. I prefer either `/commands/` or `/command-api/` as the former would be `/commands/commands`